test/library/standard/BigInteger/bigintops.graph

```diff
@@ -0,0 +1,11 @@
+perfkeys: Elapsed 64 add assign time =, Elapsed 64 pow time =, Elapsed 256 add assign time =, Elapsed 1024 add assign time =, Elapsed 256 pow time =
+files: bigintops.dat, bigintops.dat, bigintops.dat, bigintops.dat, bigintops.dat, bigintops.dat
```

Comment:
Extra .dat file?

(Note that you can use `repeat-files: bigintops.dat` when all data comes from one file to avoid having to list it out each time.)